website/comment/models.py

```diff
@@ -5,4 +5,5 @@ class comment(models.Model):
     content = models.TextField(verbose_name='评论内容')
     parent = models.ForeignKey('self', on_delete=models.CASCADE, null=True, verbose_name='父评论id')
     datetime = models.DateTimeField(auto_now_add=True, verbose_name='评论时间')
-
+    qq = models.IntegerField(verbose_name='QQ号')
+    email = models.CharField(max_length=20, verbose_name='邮箱')
```

Comment:
Have you ever tested locally?
I mean try posting some data to your locally running developing server.

Reply:
Then you'll get it that writing as this won't work as expected